content/blog/2025/251028-stac-sprint-eu-recap.md

```diff
@@ -0,0 +1,69 @@
+
+The sprint focused on fundamental questions that STAC faces as it gains popularity outside of the Earth Observation community: How do you represent multidimensional Zarr stores in STAC? What's the best way to handle variables and dimensions for datacubes? How should we navigate hierarchical Zarr structures? These aren't theoretical exercises—they're the real challenges faced by organizations working with petabytes of Earth observation data.
+
+## Key Achievements
+
+### Specification Work
+
+The sprint produced several concrete outputs that will shape how we work with multidimensional geospatial data:
+
+- **[Best Practices for STAC Zarr and N-Dimensional Arrays](https://github.com/radiantearth/stac-best-practices/pull/29)**: A comprehensive draft providing guidance on using Zarr with STAC. This is significant because, until now, nobody had proper guidance on this integration. The PR is ready for community review and needs feedback to refine and finalize the recommendations. We would like to merge it soon to provide clarity to implementers.
+
+- **[Datacube Extension Overhaul](https://github.com/stac-extensions/datacube/issue/30)**: A proposal to replace variables with bands to better align with how STAC structures metadata. This change will make the datacube extension more intuitive and powerful for multidimensional datasets. This PR will probably require further discussion before merging, as it represents a breaking change in the extension.
+
+- **[EOPF STAC Patterns](https://github.com/EOPF-Sample-Service/eopf-stac/pull/54)**: ESA-specific best practices for their Earth Observation Processing Framework (EOPF) Zarr products, ensuring that Sentinel data in Zarr format can be properly cataloged and discovered.
+
+- **[STAC in Zarr](https://github.com/radiantearth/community-sprints/blob/main/14102025-esrin-rome-italy/sprint-notes/STAC%20in%20Zarr.md)**: Explorations into embedding STAC metadata directly inside Zarr stores, making it trivial to create a STAC object directly from the Zarr metadata and ensuring that Zarr stores are entirely self-describing.
+
+{{< img src="images/251025-stac-sprint-at-work.png" caption="Participants collaborating during the sprint sessions">}}
+
+## Connecting with the Zarr Community
+
+On Friday, October 17th, STAC sprint participants were invited to join the Zarr Adopter Summit, which was happening the same week in Rome. This was a perfect opportunity to share our progress and learnings with the broader Zarr community. Several sprint participants presented the work accomplished during the week, including the proposal for a STAC convention. This cross-pollination between communities—with STAC folks learning more about Zarr developments and Zarr folks understanding STAC use cases—exemplified the collaborative spirit that makes open standards work so well.
```

Comment:
> including the proposal for a STAC convention

Can you explain what a "STAC convention" is? Just from reading this sentence, I'm not sure if it's a "STAC convention for ZARR" or "A STAC meeting aka a convention."

Reply:
It means a "Zarr convention for STAC" as in the "STAC in Zarr" bullet above. Maybe we should mention the concept of a proposed Zarr convention in that bullet.